bin/lint.rb

```diff
@@ -1515,10 +1515,10 @@ def self.fix_file(path)
         else
           # Load tests and run some quick checks:
           possible_tests.each do |test_file|
-            if !test_file.match(/-tests.yml/)
+            if !test_file.match(/[_-]tests?.ya?ml/)
```

Comment:
this regex is now identical to the regex selecting `possible_tests`, so this warning will never trigger.